src/client/connection.ts

```diff
@@ -66,7 +79,7 @@ if (fin.Window.me.uuid !== SERVICE_IDENTITY.uuid || fin.Window.me.name !== SERVI
  * @param payload Object containing additional arguments
  */
 export async function tryServiceDispatch<T extends APIFromClientTopic>(action: T, payload: APIFromClient[T][0]): Promise<APIFromClient[T][1]> {
-    const channel: ChannelClient = await channelPromise;
+    const channel: ChannelClient = (await channelPromise)!;
```

Comment:
Would it not be better for this to be `await getServicePromise`? It is probably impossible for it to actually be null, but I'd rather just use the util we have instead of asserting non-null

Reply:
Ah, yep, leftover change, fixed

Reply:
Cool!